frontend/src/components/pages/LogRecords.tsx

```diff
@@ -45,17 +47,23 @@ const mockRecords = [
     Date: "Jan 21",
     Time: "3:20 AM",
     Resident: "MB404",
-    Note: "During security check, MB404 was making some noise. TSW warned her to be quiet. She yelled on TSW behind the door, yelled, and swore (f..uck of......). TSW told her that I will call the police if she continues. Then she came down for laundry.",
+    Note:
+      "During security check, MB404 was making some noise. TSW warned her to be quiet. She yelled on TSW behind the door, yelled, and swore (f..uck of......). TSW told her that I will call the police if she continues. Then she came down for laundry.",
     Employee: "Huseyin",
     Attn_To: "John Doe",
   },
 ];
 
 const LogRecords = (): React.ReactElement => {
+  // TODO: use this instead of mockRecords & remove console.log
+  const [logRecords, setLogRecords] = useState<LogRecord[]>([]);
+  console.log(logRecords);
+
   return (
     <div className="page-container">
       <NavigationBar />
       <div className="records">
+        <SearchAndFilters setLogRecords={setLogRecords} />
```

Comment:
According to Figma it looks like there should be some padding between components here
<img width="2056" alt="Screen Shot 2023-05-21 at 11 39 34 PM" src="https://github.com/uwblueprint/supportive-housing/assets/64047863/675e10a5-5b41-4c78-9c3b-0fad0c165e18">
<img width="1071" alt="Screen Shot 2023-05-21 at 11 44 33 PM" src="https://github.com/uwblueprint/supportive-housing/assets/64047863/ef9c2f24-8e7d-4866-a9d1-9f166cdcdbe1">

Reply:
it's a gap of 12px